portfolio/src/main/java/com/google/sps/servlets/DiscussionServlet.java

```diff
@@ -1,47 +1,72 @@
-
 package com.google.sps.servlets;
-
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.util.ArrayList;
 import com.google.gson.Gson;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/discussion")
 public class DiscussionServlet extends HttpServlet {
 
-  private ArrayList<String> classicRock;
-  
   @Override
-  public void init(){
-    
-    classicRock = new ArrayList<>();
-    
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("application/json;");
```

Comment:
nit.
imo I would keep this line down to just before you send the response. Those seem like logically grouped lines, so I think they should stay near each other.

Reply:
Okay done.